Sources/Widgets/Widgets3D/PolyLineWidget/behavior.js

```diff
@@ -83,16 +83,25 @@ export default function widgetBehavior(publicAPI, model) {
     ) {
       return macro.VOID;
     }
-
-    if (model.activeState === model.widgetState.getMoveHandle()) {
+    const manipulator =
+      model.activeState?.getManipulator?.() ?? model.manipulator;
+    if (
+      model.activeState === model.widgetState.getMoveHandle() &&
+      manipulator
+    ) {
       updateMoveHandle(e);
+      const worldCoords = manipulator.handleEvent(
+        e,
+        model._apiSpecificRenderWindow
+      );
```

Comment:
isn't it already done in `updateMoveHandle()` ?

Reply:
Yes it is (see https://github.com/Kitware/vtk-js/commit/335fd1f225d36efe818a4cd2c56cbb108b75c878). Maybe we should try implement this method with all other widgets?